src/TriggerDeliveryPoliciesPage.tsx

```diff
@@ -0,0 +1,116 @@
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Button, Col, Container, ListGroup, Row, Spinner } from 'react-bootstrap';
+
+import { useAstarte } from './AstarteManager';
+import Empty from './components/Empty';
+import Icon from './components/Icon';
+import WaitForData from './components/WaitForData';
+import useFetch from './hooks/useFetch';
+import useInterval from './hooks/useInterval';
+
+interface TriggerPolicyRowProps {
+  name: string;
+  onClick: () => void;
+}
+
+const TriggerPolicyRow = ({ name, onClick }: TriggerPolicyRowProps): React.ReactElement => (
+  <ListGroup.Item>
+    <Button variant="link" className="p-0" onClick={onClick}>
+      <Icon icon="policy" className="mr-2" />
+      {name}
+    </Button>
+  </ListGroup.Item>
+);
+
+const LoadingRow = (): React.ReactElement => (
+  <ListGroup.Item>
+    <Container fluid className="text-center">
+      <Spinner animation="border" role="status" />
+    </Container>
+  </ListGroup.Item>
+);
+
+interface ErrorRowProps {
+  onRetry: () => void;
+}
+
+const ErrorRow = ({ onRetry }: ErrorRowProps): React.ReactElement => (
+  <ListGroup.Item>
+    <Empty title="Couldn't load available delivery policies" onRetry={onRetry} />
+  </ListGroup.Item>
+);
+
+export default (): React.ReactElement => {
+  const astarte = useAstarte();
+  const navigate = useNavigate();
+  const policiesFetcher = useFetch(astarte.client.getTriggerDeliveryPolicyNames);
+
+  useInterval(policiesFetcher.refresh, 30000);
+
+  return (
+    <Container fluid className="p-3" data-testid="policies-page">
+      <Row>
+        <Col>
+          <h2>Trigger Delivery Policies</h2>
+        </Col>
+      </Row>
+      <Row className="mt-3">
+        <Col sm={12}>
+          <ListGroup>
+            <ListGroup.Item>
+              <Button
+                variant="link"
+                className="p-0"
+                onClick={() => {
+                  navigate('/trigger-delivery-policies/new');
+                }}
+              >
+                <Icon icon="add" className="mr-2" />
+                Install a new trigger delivery policy...
+              </Button>
+            </ListGroup.Item>
+            <WaitForData
+              data={policiesFetcher.value}
+              status={policiesFetcher.status}
+              fallback={<LoadingRow />}
+              errorFallback={<ErrorRow onRetry={policiesFetcher.refresh} />}
+            >
+              {(policies) => (
+                <>
+                  {policies.map((policy) => (
+                    <TriggerPolicyRow
+                      key={policy}
+                      name={policy}
+                      onClick={() => {
+                        navigate(`/trigger-delivery-policies/${policy}/edit`);
```

Comment:
The route for this link does not exist.
To avoid having a broken functionality, it would be great to expand this pull request and add a page to review the definition (read-only) of an existing trigger delivery policy. See the TriggerPage component for an example on how to fetch and display the definition of a resource: https://github.com/astarte-platform/astarte-dashboard/blob/5064e75bbb2f67b346290733ddcc40ac3226b3fb/src/TriggerPage.tsx
The page should also have a button to delete the existing policy, which will display a confirmation modal before attempting the deletion: see again the TriggerPage component for an example of a delete modal: https://github.com/astarte-platform/astarte-dashboard/blob/5064e75bbb2f67b346290733ddcc40ac3226b3fb/src/TriggerPage.tsx#L39